setup.py

```diff
@@ -5,15 +5,18 @@
     name='pykeepass',
     version='2.8.2',
     license='GPL3',
-    description='Low-level library to interact with keepass databases '
-                '(supports the v.4 format)',
+    description='Python library to interact with keepass databases '
+                '(supports KDBX3 and KDBX4)',
     long_description=open('README.rst').read(),
     author='Philipp Schmitt',
     author_email='philipp@schmitt.co',
     url='https://github.com/pschmitt/pykeepass',
-    packages=find_packages(),
-    install_requires=['libkeepass', 'easypysmb', 'python-dateutil'],
-    entry_points={
-        'console_scripts': ['pkpwrite=pykeepass.pkpwrite:main']
-    }
+    packages=['pykeepass'],
+    install_requires=[
```

Comment:
The libkeepass_proto dependency is missing here.

Reply:
It's all in `kdbx_parsing/`

Reply:
Oh okay - sorry, I should have actually read your description -_-'

+1 for deprecating libkeepass, in retrospect I should have done that a long time ago.

Reply:
@Evidlo what features are missing ?

Reply:
Twofish support and leaving entries protected (entries are always unprotected right now).

Also since the KDBX4 format shuffled things around, the location of certain low-level attribute changes.

For example, to access the protected stream key:

in KDBX3: `kp.kdbx.header.value.dynamic_header.protected_stream_key.data`
in KDBX4: `kp.kdbx.body.payload.inner_header.protected_stream_key.data`

Reply:
Had any time to look at this?